app/views/hiring_staff/schools/_vacancy.html.haml

```diff
@@ -1,5 +1,5 @@
 %tr.vacancy[vacancy]
-  %td= link_to vacancy.job_title, school_job_path(vacancy.id), class: 'view-vacancy-link'
+  %td= link_to vacancy.job_title, job_path(vacancy), class: 'view-vacancy-link'
```

Comment:
This changes existing functionality and removes access to the admin job page.
The dashboard should be extended to cater for displaying (or copying to clipboard maybe?) the public job listing url - and only for published jobs as we should never link to non existing links.